### ISSUE TITLE
Possible explanation to the unstable behavior of SCSCNN in training

### ISSUE THREAD

**Issue by D0048** [MyWorkShop/Convolutional-Neural-Networks-in-Convolution#8]
@mwsht 
According to this [article](https://arxiv.org/pdf/1406.2572v1.pdf), the gradient of SCSCNN model will be reeeeeeeeeally flat and full of saddle points...

Maybe consider a alternate training method?

**Comment by D0048** [comment]
This also suggest possible trails with larger learning rate with smaller batch sizes. According to my test, learning rate `0.1` works just fine for converging into an accuracy of 96%--and drops to 80% in one epoch--which doubted to be an overshoot. 

Stochastic Gradient Descent may be worth trying.

Further tests pending.
